src/librustdoc/html/render.rs

```diff
@@ -1289,8 +1289,9 @@ fn settings(root_path: &str, suffix: &str) -> String {
             .into(),
         ("auto-hide-attributes", "Auto-hide item attributes.", true).into(),
         ("auto-hide-method-docs", "Auto-hide item methods' documentation", false).into(),
-        ("auto-hide-trait-implementations", "Auto-hide trait implementations documentation", true)
+        ("auto-hide-trait-implementations", "Auto-hide trait implementation documentation", true)
             .into(),
+        ("auto-collapse-implementors", "Auto-hide the types which implement a trait", true).into(),
```

Comment:
"implementors of a trait"

Reply:
Oh that's funny, I just asked guillaume to change it from that to the current text. Why do you think 'implementors' is more clear?

Reply:
It's more concise without being ambiguous. Typically you don't want to use compound clauses in a setting name, it should be short but descriptive